terraform/azure/projects/utoronto.tfvars

```
@@ -0,0 +1,19 @@
+subscription_id    = "ead3521a-d994-4a44-a68d-b16e35642d5b"
+resourcegroup_name = "2i2c-utoronto-cluster"
+
+ssh_pub_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQJ4h39UYNi1wybxAH+jCFkNK2aqRcuhDkQSMx0Hak5xkbt3KnT3cOwAgUP1Vt/SjhltSTuxpOHxiAKCRnjwRk60SxKhUNzPHih2nkfYTmBBjmLfdepDPSke/E0VWvTDIEXz/L8vW8aI0QGPXnXyqzEDO9+U1buheBlxB0diFAD3vEp2SqBOw+z7UgrGxXPdP+2b3AV+X6sOtd6uSzpV8Qvdh+QAkd4r7h9JrkFvkrUzNFAGMjlTb0Lz7qAlo4ynjEwzVN2I1i7cVDKgsGz9ZG/8yZfXXx+INr9jYtYogNZ63ajKR/dfjNPovydhuz5zQvQyxpokJNsTqt1CiWEUNj georgiana@georgiana"
+
+global_container_registry_name = "2i2cutorontohubregistry"
+global_storage_account_name = "2i2cutorontohubstorage"
+
+location" = "Canada Central"
```

The default location for the other clusters is `westus2`:
https://github.com/2i2c-org/infrastructure/blob/852afb497ab42dda375fc22bda551b9afbafe97e/terraform/azure/variables.tf#L21

---

```suggestion
location = "canadacentral"
```

We need the `Name` rather than `DisplayName`. See `az account list-locations -o table` using the Azure CLI.

---

Oh, right! Thank you :sparkles: